scss/components/_menu.scss

```diff
@@ -52,6 +52,11 @@ $menu-item-background-hover: $light-gray !default;
 
   li {
     @include disable-mouse-outline;
+
+    @if $global-flexbox {
```

Comment:
Not sure I understand the point of this, could you explain? If we are adding spacing between `li` items, it should be at least controllable by the user. Why not have them sitting flush together though?